### ISSUE TITLE
Add WASD Scrolling

### ISSUE THREAD

**Issue by Phlekray** [zocker-160/Die-Siedler-Aufbruch-der-Kulturen-widescreen-fix#1]
Is it possible to add WASD scrolling instead of the arrow keys? Or customizable key bindings?

**Comment by zocker-160** [comment]
Yes to both questions, however while the former should be possible using a config file setting for my dll the latter would require some significant effort including creating some kind of GUI tool to set the new keys.

**Comment by Phlekray** [comment]
So this is already possible with the existing dll? Could you add an example config file for WASD scrolling?

**Comment by zocker-160** [comment]
No it is not possible currently, I meant that it could be made by simply adding an option to the config.

**Comment by Phlekray** [comment]
Ah, okay, would be a great feature!

**Comment by zocker-160** [comment]
I took a look and sadly this is not going to be easy since the "S" key is already bound to opening the statistics screen. 

So in order to make this work, I would need to rebind existing keys as well. I will keep investigating, but I cannot promise since I am currently short on time sadly.
